pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1181,7 +1181,7 @@ class Timedelta(_Timedelta):
 
             kwargs = {key: _to_py_int_float(kwargs[key]) for key in kwargs}
 
-            nano = kwargs.pop('nanoseconds', 0)
+            nano = np.timedelta64(kwargs.pop('nanoseconds', 0))
```

Comment:
could/should this have the unit 'ns' made explicit?

Reply:
Looking at the broader code block, it seems like the idea is to make sure the arguments can be used to construct a `Timedelta` at the highest precision unit, `ns`. 

Looks like the subsequent `convert_to_timedelta64` forces a `ns` cast in any case, and I think `nano` will get subject to that cast automatically as a generic `m8` too, right?

I don't see `nano` variable used for anything else beyond that sanity check.

Reply:
I'd be +1 on including the unit in this expression

Reply:
ok, done